javagen/src/main/java/com/azure/autorest/util/MethodUtil.java

```diff
@@ -98,6 +101,25 @@ public static Request tryMergeBinaryRequests(List<Request> requests, Operation o
         return selectedRequest;
     }
 
+    /**
+     * build the string of allowed values for an enum type parameter
+     * @param parameter an enum type parameter
+     * @return the string of allowed values for this enum type parameter
+     */
+    public static String buildAllowedEnumValues(Parameter parameter) {
+        if (parameter.getSchema() == null || !(Mappers.getSchemaMapper().map(parameter.getSchema()) instanceof EnumType)) {
+            return "";
+        }
+        String res = "Allowed values: ";
+        EnumType enumType = (EnumType) Mappers.getSchemaMapper().map(parameter.getSchema());
+        List<ClientEnumValue> choices = enumType.getValues();
+        if (choices != null && !choices.isEmpty()) {
+            res += choices.stream().map(choice -> choice.getValue()).collect(Collectors.joining(", "));
```

Comment:
Should we quote the value, if it is string?

What if enum contains ","?

What if enum contains "<"?

What if enum is "@abc"?

Did you handle all of them (maybe more) correctly?